src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -11,176 +11,199 @@ import {
   type UtxoChainId,
   utxoChainIds,
 } from '@shapeshiftoss/chain-adapters'
+import type { HDWallet } from '@shapeshiftoss/hdwallet-core'
 import { supportsETH } from '@shapeshiftoss/hdwallet-core'
 import type { KnownChainIds } from '@shapeshiftoss/types'
+import { useCallback } from 'react'
 import { useTranslate } from 'react-polyglot'
-import { useSelector } from 'react-redux'
 import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
-import { useEvm } from 'hooks/useEvm/useEvm'
+import { getSupportedEvmChainIds } from 'hooks/useEvm/useEvm'
```

Comment:
We didn't need to import a hook just to get this, made it a pure function instead